importers/aircraft_csv_format_importer.py

```diff
@@ -0,0 +1,155 @@
+        else:
+            # Skip line if invalid timestamp
+            self.errors.append(
+                {self.error_type: f"Line {line_number}. Error in timestamp parsing."}
+            )
+            return
+
+        # And finally store it - Check how to do this as currently don't have a platform name
+        platform = self.get_cached_platform(data_store, platform_name=None, change_id=change_id)
+        sensor_type = data_store.add_to_sensor_types("Location-Satellite", change_id=change_id).name
+        privacy = get_lowest_privacy(data_store)
+        sensor = platform.get_sensor(
+            data_store=data_store,
+            sensor_name="GPS",
+            sensor_type=sensor_type,
+            privacy=privacy,
+            change_id=change_id,
+        )
+        state = datafile.create_state(data_store, platform, sensor, timestamp, self.short_name)
+
+        # Set the location conversion
+        location = Location(errors=self.errors, error_type=self.error_type)
+        print(lat_degrees_token.text)
+        print(long_degrees_token.text)
+        lat_success = location.set_latitude_decimal_degrees(lat_degrees_token.text)
+        lon_success = location.set_longitude_decimal_degrees(long_degrees_token.text)
+        if lat_success and lon_success:
+            state.location = location
+            combine_tokens(long_degrees_token, lat_degrees_token).record(
+                self.name, "location", state.location, "decimal degrees"
+            )
+        else:
+            self.errors.append({self.error_type: f"Line {line_number}. Error in Location Parsing."})
+
+        # Convert the altitude and check valid
+        elevation_valid, elevation = convert_distance(
+            altitude_token.text, unit_registry.feet, line_number, self.errors, self.error_type
+        )
+        if elevation_valid:
+            state.elevation = elevation
+            altitude_token.record(self.name, "altitude", state.elevation)
+
+        # Convert the speed and check valid
+        speed_valid, speed = convert_speed(
+            speed_token.text,
+            unit_registry.knots,
+            line_number,
+            self.errors,
+            self.error_type,
+        )
+        if speed_valid:
+            state.speed = speed
+            speed_token.record(self.name, "speed", speed)
+
+        # Convert the course into Rads and check valid
+        course_valid, course = convert_absolute_angle(
+            course_token.text, line_number, self.errors, self.error_type
+        )
+        if course_valid:
+            state.course = course
+            course_token.record(self.name, "course", course)
+
+    @staticmethod
+    def parse_timestamp(date, time):
+        format_str = "%d/%m/%Y "
+        format_str += "%H:%M:%S"
+
+        try:
+            res = datetime.strptime(date.strip() + " " + time.strip(), format_str)
+        except ValueError:
```

Comment:
This bit was missing coverage too, but should be caught by the test I suggested adding in one of my other comments - for something like a time of 12:99:30 or 99:01:02.